Object.cpp

```diff
@@ -226,7 +250,25 @@ class SamsungA5 : public Samsung , public phoneSetting{
         SamsungA5():Samsung(22000,22,"Dual SIM","Samsung","SamsungA5","12987387"){}
         phoneSetting *setting = phoneSetting::getInstance();
            
-        
+         void orderFood()
```

Comment:
This Method is being repeated multiple times. Follow DRY principle to reuse the method for some interface.